backend/services/implementations/authService.ts

```diff
@@ -160,6 +160,23 @@ class AuthService implements IAuthService {
     }
   }
 
+  async resetPasswordCode(email: string): Promise<string> {
+    try {
+      const resetLink = await firebaseAdmin
```

Comment:
`resetLink` is of the form https://example.com/usermgmt?mode=resetPassword&oobCode=ABC123&apiKey=AIzaSy...&lang=fr